clang/lib/Format/Format.cpp

```diff
@@ -958,9 +958,13 @@ template <> struct MappingTraits<FormatStyle> {
       // For backward compatibility.
       BracketAlignmentStyle LocalBAS = BAS_Align;
       if (IsGoogleOrChromium) {
-        if (Style.Language == FormatStyle::LK_JavaScript) {
+        FormatStyle::LanguageKind Language =
```

Comment:
Based on what you said: Shouldn't this only be done if `Style.Language == LK_None`? What if (like in the previously failing test) `Language` is set to Java or JavaScript, but we parse explicitly another language?

Reply:
Should we have a test for that scenario?